libmogwai-schedule/scheduler.c

```diff
@@ -235,6 +239,29 @@ mws_scheduler_class_init (MwsSchedulerClass *klass)
                            MWS_TYPE_PEER_MANAGER,
                            G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY | G_PARAM_STATIC_STRINGS);
 
+  /**
+   * MwsScheduler:allow-downloads:
+   *
+   * Whether any of the currently active network connections are configured to
+   * allow any large downloads. This is not a guarantee that a schedule entry
+   * will be scheduled; it is a reflection of the user’s intent for the use of
+   * the currently active network connections, intended to be used in UIs to
+   * remind the user of how they have configured the network.
+   *
+   * Programs must not use this value to check whether to schedule an entry.
+   * Schedule the entry unconditionally; the scheduler will work out whether
+   * (and when) to download the entry.
+   *
+   * Since: 0.1.0
+   */
+  props[PROP_ALLOW_DOWNLOADS] =
+      g_param_spec_boolean ("allow-downloads", "Allow Downloads",
+                            "Whether any of the currently active network "
+                            "connections are configured to allow any large "
```

Comment:
I'd replace _large_ by _background_ here and in the docs above, just because it keeps me wondering 'what's "large"?'.

Reply:
Hmm, I’ll have a think about this. Mogwai doesn’t have to be used for downloads in the background, though; it could be used in the foreground.